tests/integration/fixtures/mini_sentry.py

```diff
@@ -12,6 +14,19 @@
 from . import SentryLike
 
 
+# HACK: import the envelope module from libsemaphore without requiring to build the cabi
+__envelope = types.ModuleType("envelope")
```

Comment:
Why do we need to construct this? Can't we just pass a regular dict as namespace to eval and read the items out of there?

Reply:
You tell me, should we? :)

Reply:
Seems to work, pushed